spec/spec_helper.rb

```diff
@@ -11,35 +12,29 @@
 require 'napa'
 require 'napa/rspec_extensions/response_helpers'
 
+module NapaSpecClassHelpers
+  # Apparently Thor's last name is Odinson... who knew?!
+  # http://marvel.com/universe/Thor_(Thor_Odinson)
+  # http://www.infinitelooper.com/?v=4D7cPH7DHgA#/170;230
+  def tell_me_mr_odinson_what_good_is_a_phone_call_if_youre_unable_to_speak?
+    before do |spec|
+      allow_any_instance_of(Thor::Shell::Basic)
+        .to receive(:stdout).and_return(object_spy $stdout)
+      allow_any_instance_of(Thor::Shell::Basic)
+        .to receive(:stderr).and_return(object_spy $stderr)
+    end
+  end
+end
+
+ActiveRecord::Schema.verbose = false
+
 # from https://gist.github.com/adamstegman/926858
 RSpec.configure do |config|
   config.include Napa::RspecExtensions::ResponseHelpers
-
-  config.before(:all) { silence_output }
-  config.after(:all) { enable_output }
-
+  config.extend  NapaSpecClassHelpers
   config.include ActsAsFu
 
   config.before(:each) do
-    allow(Napa).to receive(:initialize)
```

Comment:
1. It didn't do anything: there's an if statement around the whole thing [here](https://github.com/bellycard/napa/blob/7350ee95cca1cb3efd0e5ec111d636b35be57273/lib/napa.rb#L51), and it's set to skip the if statement [here](https://github.com/bellycard/napa/blob/7350ee95cca1cb3efd0e5ec111d636b35be57273/spec/spec_helper.rb#L9). 
2. Ruby freaks out a little bit when you remove initialize `$ ruby -e 'Class.new { alias initialize to_s; remove_method :initialize }'` output: `-e:1: warning: removing 'initialize' may cause serious problems` which [rspec does](https://github.com/rspec/rspec-mocks/blob/56392d9223869704852706829e65661106f0495f/lib/rspec/mocks/method_double.rb#L261) as part of the removal of its stubs.